src/routes/expenses.router.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const controller = require('../controllers/expenses.controller');
+
+const expensesRouter = express.Router();
+
+expensesRouter.get('/', express.json(), controller.getAllExpenses);
+
+expensesRouter.get('/:id', express.json(), controller.getExpense);
+
+expensesRouter.post('/', express.json(), controller.post);
+
+expensesRouter.delete('/:id', express.json(), controller.removeExpense);
+
+expensesRouter.patch('/:id', express.json(), controller.updateExpense);
```

Comment:
you can just write `express.json()` once in createServer.js, where you use this module

Reply:
Fix this